docs/workflow/building/libraries/README.md

```diff
@@ -117,7 +117,7 @@ When building the repo with any native sanitizers, you should build all native c
 
 ### How to build native components only
 
-The libraries build contains some native code. This includes shims over libc, openssl, gssapi, and zlib. The build system uses CMake to generate Makefiles using clang. The build also uses git for generating some version information.
+The libraries build contains some native code. This includes shims over libc, openssl, gssapi, and zlib-ng. The build system uses CMake to generate Makefiles using clang. The build also uses git for generating some version information.
```

Comment:
```suggestion
The libraries build contains some native code. This includes shims over libc, openssl, gssapi, and zlib. The build system uses CMake to generate Makefiles using clang. The build also uses git for generating some version information.
```
I think it is more appropriate to just say zlib here. The shim can be for system zlib or zlib-ng.

Similarly, we just say libc here even though it can be glibc, musl-libc or bionic-libc.